src/components/workspace/controlBar/chapterSelect.tsx

```diff
@@ -38,25 +39,41 @@ export function ChapterSelect(props: ChapterSelectProps) {
   );
   const ChapterSelectComponent = Select.ofType<IChapter>();
 
+  const chapterSelector = (currentChap: number, currentVariant: Variant, disabled: boolean) => {
+    return (
+      <div>
+        <Button
+          className={Classes.MINIMAL}
+          text={styliseChapter(currentChap, currentVariant)}
+          rightIcon={disabled ? null : IconNames.DOUBLE_CARET_VERTICAL}
+          disabled={disabled}
```

Comment:
`disabled={disabled || false}` here, so that Sourcereel, Sourcecast etc don't need to be changed.

Reply:
Aight Aight